Sources/Sentry/Transactions/SentrySpanContext.h

```diff
@@ -0,0 +1,37 @@
+#import "SentryDefines.h"
+#import "SentrySerializable.h"
+#import <Sentry/SentrySpanId.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class SentryId;
+
+NS_SWIFT_NAME(SpanContext)
+@interface SentrySpanContext : NSObject
+
+@property (nonatomic, strong) SentryId *traceId;
+
+@property (nonatomic, strong) SentrySpanId *spanId;
+
+@property (nonatomic, strong) SentrySpanId *_Nullable parentSpanId;
```

Comment:
we could call them `SpanId` only instead of `SentrySpanId`, Java/.NET is like this

Reply:
I believe, like the previous comment must keep the prefix `Sentry`. Another one I'll close but against feel free to reopen if strong disagree. I pinged @mitsuhiko to confirm on this but afaik these types need to be prefixed. @HazAT maybe can chime in